provider/info/info_data_source.go

```diff
@@ -0,0 +1,151 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package info
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-framework/datasource"
+	"github.com/hashicorp/terraform-plugin-framework/datasource/schema"
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	sdk "github.com/openshift-online/ocm-sdk-go"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/accountsmgmt/v1"
+)
+
+type OCMInfoDataSource struct {
+	collection      *cmv1.CurrentAccountClient
+	ocmAPI          string
+	ocmAwsAccountID string
+}
+
+var _ datasource.DataSource = &OCMInfoDataSource{}
+var _ datasource.DataSourceWithConfigure = &OCMInfoDataSource{}
+
+func New() datasource.DataSource {
+	return &OCMInfoDataSource{}
+}
+
+func (d *OCMInfoDataSource) Metadata(ctx context.Context, req datasource.MetadataRequest, resp *datasource.MetadataResponse) {
+	resp.TypeName = req.ProviderTypeName + "_info"
+}
+
+func (d *OCMInfoDataSource) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Description: "",
+		Attributes: map[string]schema.Attribute{
+			"account_id": schema.StringAttribute{
+				Description: "OCM account ID",
+				Computed:    true,
+			},
+			"account_name": schema.StringAttribute{
+				Description: "OCM account User full name",
+				Computed:    true,
+			},
+			"account_username": schema.StringAttribute{
+				Description: "OCM account username",
+				Computed:    true,
+			},
+			"account_email": schema.StringAttribute{
+				Description: "OCM account email",
+				Computed:    true,
+			},
+			"organization_id": schema.StringAttribute{
+				Description: "OCM account organization id",
+				Computed:    true,
+			},
+			"organization_external_id": schema.StringAttribute{
+				Description: "OCM account organization external id",
+				Computed:    true,
+			},
+			"organization_name": schema.StringAttribute{
+				Description: "OCM account organization name",
+				Computed:    true,
+			},
+			"ocm_api": schema.StringAttribute{
+				Description: "OCM API url",
+				Computed:    true,
+			},
+			"ocm_aws_account_id": schema.StringAttribute{
+				Description: "OCM AWS account ID",
```

Comment:
It's a little bit confusing- what is the difference between the "account_id" attribute -
In my opinion 
account_id -> ocm_account_id
ocm_aws_account_id   -> user_account_id

Reply:
OK, So just update the description